python/utils/__init__.py

```diff
@@ -1,5 +1,9 @@
 from utils import colorText, str2bool, find_moose_executable, runExe, check_configuration
-from MooseYaml import MooseYaml
+
+try:
+    from MooseYaml import MooseYaml
+except:
+    pass
```

Comment:
I guess I don't understand how this is optional. Maybe if it's not there we set a Boolean and report a better error if we attempt to use the library?